packages/web_benchmarks/test/src/browser_test.dart

```diff
@@ -0,0 +1,55 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+
+import 'package:web_benchmarks/src/browser.dart';
+
+import 'browser_test_json_samples.dart';
+
+void main() {
+  group('BlinkTraceEvent works with Chrome 89+', () {
```

Comment:
Shouldn't we have tests for the older versions that are still supported in the code as well?

Reply:
It's fine to only test 89+. We only support the last 2 versions of each browser.

Reply:
I've modified the cirrus script to add a matrix here so the benchmark test gets run both with Chrome 84 (old behavior) and Chrome 89 (new behavior).

(Unit tests are probably running in the VM.)

Reply:
Now we have a benchmark exercising the old version of the code (even though we don't have a unit test specifically for the frames returned by the old version of the browser). I think this better reflects the guarantees we're doing about that specific bit of code :P